lib/rubygems/server.rb

```diff
@@ -431,7 +431,8 @@ def initialize(gem_dirs, port, daemon, launch = nil, addresses = nil)
     begin
       require 'webrick'
     rescue LoadError
-      abort "webrick is not found. You may need to `gem install webrick` to install webrick."
+      puts "webrick is not found. You may need to `gem install webrick` to install webrick."
+      exit
```

Comment:
Here we're undoing a patch merged directly to the ruby-core master branch without contributing upstream. It can be manually removed from this PR but it'd be nice if we could get that change contributed upstream :+1:.

Reply:
Ah, I forgot to submit this to rubygems/rubygems.